lib/home/widgets/settings_screen.dart

```diff
@@ -16,6 +19,27 @@ class SettingsScreen extends StatelessWidget {
                 icon: const Icon(Icons.help))
           ],
         ),
-        body: const Center(child: Text('No settings currently available')));
+        body: const LoginTile());
+  }
+}
+
+class LoginTile extends StatelessWidget {
+  const LoginTile({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<LoginBloc, LoginState>(builder: (context, state) {
+      return Padding(
+        padding: const EdgeInsets.all(4.0),
+        child: ListTile(
+          leading: const Icon(Icons.account_circle),
+          title: const Text('Edit user information'),
+          subtitle: Text(
+              'Employee Name: ${state.loginInfo.employeeName} \nEmployee ID: ${state.loginInfo.employeeId}'),
+          onTap: () =>
+              {Navigator.pushReplacementNamed(context, PageRouter.login)},
```

Comment:
As @akabla said, in the future we might add some more expressive UI to make this seem more editable, but for now it seems good to me!